_posts/2016-02-29-phishing.md

```diff
@@ -0,0 +1,54 @@
+
+"Phishing" attacks are attempts to acquire information such as usernames, passwords, and credit card details by masquerading as a trustworthy entity. The name derives from the similarity of using a bait in an attempt to catch a victim from a pool and the "bait" normally consist of an electronic message such as an email sent to a large group with the hope that some small percentage of targeted victims will fall for the ploy.
+
+A phishing email is generally sent by a malicious source but purporting to be from a known or trusted source such as a company's IT department or a popular social media website or high street bank. Phishing emails will often either have a malicious payload such as a word document with a macro, or else contain links to websites that are either infected with malware or attempt to trick people into entering their credentials.
+
+Typically the email itself, or the website linked to from the email use corporate imagery, branding and fonts to attempt to provide a look and feel are almost identical to the genuine, legitimate business being impersonated.
+
+
+Phishing attacks are difficult to entirely prevent against, but are a good example of a situation in which "defense in depth" is appropriate - no one measure on its own is effective, but by implementing a range of anti-phishing countermeasures, the effectiveness of phishing attacks can be greatly reduced. So what can be done?
+
+### 1. Server-side Email filtering
+
+The first line of defense needed is anti-spam and email filtering for inbound email, to prevent emails from being received by your employees in the first place. Various technologies can be used including anti-spam filter based on BLs and IP reputation services, a server-side antivirus scan for attachments, and MX record/domain verification using Sender Policy Framework (SPF). Emails can be rejected by your mailserver so that they never make it to your users.
+
+### 2. User training and internal policies
+
+Inevitably, some emails are going to get through your email filters, no matter how infrequently. It is important to reduce the likelihood that employees are tricked into falling for the phishing scams by providing training as to how to recognise likely phishing attempts. It helps if advice is consistent - if users are aware that internal IT staff will never ask them for their passwords, then they are less likely to fall for a phishing attack asking them for their password. There are also excellent online tools such as [VirusTotal](https://www.virustotal.com/) which allow users to upload suspected attachments before opening them in order to check whether they contain any viruses.
+
+### 3. Mail client software
+
+Mail client software has a part to play in providing warning to users that opening attachments from emails or clicking on links may provide some element of risk of virus and phishing attacks and provide reminders to users of their training at the time that the phishing email is received.
```

Comment:
Could this be broken up into smaller sentences? It's a bit long and awkward at the moment